tests/gcloud-testing-core/src/main/java/com/google/cloud/runtime/jetty/testing/AppDeployment.java

```diff
@@ -0,0 +1,57 @@
+package com.google.cloud.runtime.jetty.testing;
+
+import org.eclipse.jetty.toolchain.test.MavenTestingUtils;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Objects;
+
+public final class AppDeployment {
```

Comment:
Added as part of Issue #7 - https://github.com/GoogleCloudPlatform/jetty-runtime/commit/af0a64ea6fc849020d0af4188fdb590bb81e19d5